src/fvOptions/actuatorLineSource/actuatorLineElement/profileData/profileData.C

```diff
@@ -494,7 +507,7 @@ Foam::List<scalar> Foam::profileData::normalCoefficientList
             newList.append
             (
                 liftCoefficientList_[i]*cos(degToRad(angleOfAttackList_[i]))
-              - dragCoefficientList_[i]*sin(degToRad(angleOfAttackList_[i]))
+              + dragCoefficientList_[i]*sin(degToRad(angleOfAttackList_[i]))
```

Comment:
I was thinking maybe here we could simply call

```c++
    newList.append(normalCoefficient(angleOfAttackList_[i]);
```

so the same calculation is used, though this will call interpolation functions for lift and drag, so maybe it's not worth the additional expense.